### ISSUE TITLE
Ärzte im Nur-Notfall-Modus

### ISSUE THREAD

**Issue by Matthias84** [binary-butterfly/shared-delivery#44]
Derzeit kümmern sich die meisten Ärzte nur noch um akute Notfälle. Leider werden existieren Termine nicht abgesagt, weshalb Patienten entweder trotzdem hingehen (und weggeschickt werden), oder dauernd anrufen.

Wenn man für Ärzte vermerken könnte, dass nur dringende Sachen behandelt werden, wäre das super.

**Comment by the-infinity** [comment]
Interessante und sinnvolle Idee. Gibts dafür schon ein OSM Tag?

**Comment by Matthias84** [comment]
Es gäbe emergency=yes, was man neu einführen müsste als emergency:covid19=yes|only ?

**Comment by the-infinity** [comment]
Da alles auf lokalwirkt covid19-bezogen ist das :covid19 ja erstmal irrelevant (aber langfristig würde ich natürlich schon gerne mehr aus der Seite machen. Lies: Kommunen dauerhaft als OSM Datenlieferant anzapfen). 

Das yes|only Konstrukt zu übernehmen ist eine sehr gute Idee, mache ich so.

**Comment by the-infinity** [comment]
Hmm. Beim Implementieren ist mir aufgefallen: was unterscheidet denn "emergency: yes" von keinem Emergency? Only ist klar, da wäre die Aussage "hat ausschließlich für Notfälle offen". Aber "yes"?

**Comment by Matthias84** [comment]
Yes kommt insb. aus der Nutzung als klassisches Tag z.B. für eine Klinik welche auch Notfall-Behandlungen durchführt. Für Covid19 reicht sicherlich only.
